catalog.html

```diff
@@ -92,241 +92,286 @@ <h1 class="page__title">
-                                    <span class="form-checkbox__box">
-                                    </span>
-                                    <span class="form-checkbox__text">
-                                        Combatant  
-                                    </span>
-                                </p>
-                            </label>
-                            <label class="form-checkbox">
-                                <input type="checkbox" class="input-hidden form-checkbox__input">
-                                <p class="form-checkbox__inner">  
-                                    <span class="form-checkbox__box">
-                                    </span>
-                                    <span class="form-checkbox__text">
-                                        Ninja  
-                                    </span>
-                                </p>
-                            </label>
-                            <label class="form-checkbox">
-                                <input type="checkbox" class="input-hidden form-checkbox__input">
-                                <p class="form-checkbox__inner">  
-                                    <span class="form-checkbox__box">
-                                    </span>
-                                    <span class="form-checkbox__text">
-                                        King  
-                                    </span>
-                                </p>
-                            </label>
-                            <label class="form-checkbox">
-                                <input type="checkbox" class="input-hidden form-checkbox__input">
-                                <p class="form-checkbox__inner">  
-                                    <span class="form-checkbox__box">
-                                    </span>
-                                    <span class="form-checkbox__text">
-                                        Queen  
-                                    </span>
-                                </p>
-                            </label>
-                            <label class="form-checkbox">
-                                <input type="checkbox" class="input-hidden form-checkbox__input">
-                                <p class="form-checkbox__inner">  
-                                    <span class="form-checkbox__box">
-                                    </span>
-                                    <span class="form-checkbox__text">
-                                        Robot  
-                                    </span>
-                                </p>
-                            </label>
+                            <div>
+                                <label class="form-checkbox">
+                                    <input type="checkbox" name="" class="input-hidden form-checkbox__input" name="clothing" value="combatant">
```

Comment:
You can reduce your classes. For example:
form-checkbox => checkbox
form-radio => radio

Reply:
Done